components/server/src/user/user-service.ts

```diff
@@ -229,12 +292,18 @@ export class UserService {
     async getWorkspaceUsageAttributionId(user: User, projectId?: string): Promise<string | undefined> {
         // A. Billing-based attribution
```

Comment:
TODO: Handle UBB flag off (fall back to Project-based attribution? or always to user?)